timeless/reservations/views.py

```diff
@@ -18,13 +18,38 @@
 
 
 class ReservationsListView(views.ListView):
-    """ List the reservation """
+    """ List the reservations """
     model = Reservation
     template_name = "reservations/list.html"
-    context_object_list_name = "reservations"
 
 
-ReservationsListView.register(BP, "/")
+class ReservationsViewCreate(views.CreateView):
+    """ Create a new reservation instance"""
+    template_name = "reservations/create_edit.html"
+    success_view_name = "reservations.list"
+    form_class = ReservationForm
+
+
+class ReservationsEditView(views.UpdateView):
+    """Update reservation"""
+    template_name = "restaurants/create_edit.html"
+    form_class = ReservationForm
+    model = Reservation
+    success_view_name = "reservations.list"
+
+
+class ReservationsDeleteView(views.DeleteView):
+    """Delete reservation
+    Deletes location using id and redirects to list page
+    """
+    success_view_name = "reservations.list"
+    model = Reservation
+
+
+ReservationsListView.register(BP, "/", name="list")
+ReservationsViewCreate.register(BP, "/create", name="create")
+ReservationsEditView.register(BP, "/edit/<int:id>", name="edit")
+ReservationsDeleteView.register(BP, "/delete/<int:id>", name="delete")
```

Comment:
@vladarefiev I don't think other endpoints were given a ``name`` when being registered -- what is it for?

Reply:
@amihaiemil It's for providing a names in namespace for them. Sometimes when people didn't it, therefore I'm fixing their tests right now ;)